### ISSUE TITLE
gstreamer1.0-doc has no installation candidate on Debian

### ISSUE THREAD

**Issue by donewiththedollar** [flet-dev/flet#1137]
**Description**

**Code example to reproduce the issue:**

``sudo apt-get install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 gstreamer1.0-qt5 gstreamer1.0-pulseaudio
``

**Describe the results you received: `Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package gstreamer1.0-doc is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
`**

**Describe the results you expected:** The package should be available on Debian no? Is there some repository that is missing by default on Debian or was this package actually removed and should be removed from the documentation?

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.4.2
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page: None
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Location: /home/bitnami/.local/lib/python3.9/site-packages
Requires: websockets, httpx, websocket-client, watchdog, flet-core, packaging, oauthlib
Required-by: 
```

**Operating system**: Linux ip-172-26-11-0 5.10.0-21-cloud-amd64 #1 SMP Debian 5.10.162-1 (2023-01-21) x86_64 GNU/Linux

**Additional environment details:**

**Comment by FeodorFitsner** [comment]
It should be available on Debian, yes: https://gstreamer.freedesktop.org/documentation/installing/on-linux.html?gi-language=c#install-gstreamer-on-ubuntu-or-debian

**Comment by donewiththedollar** [comment]
> 

I see that but the thing is I ran ``apt-get install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 gstreamer1.0-qt5 gstreamer1.0-pulseaudio`` as described in the documentation and the command fails.

Perhaps documentation needs to be updated?

**Comment by FeodorFitsner** [comment]
Could it be that GStreamer is not available for Debian 5.1 you have?
